src/main/java/api/image/ImageConverter.java

```diff
@@ -1,9 +1,6 @@
 package api.image;
-
 import java.awt.*;
-
```

Comment:
These spaces are needed :)

Reply:
Please, use Reformat Code action if you are using IntelliJ IDEA

Reply:
+